core/src/main/kotlin/utilities/safeEnumValueOf.kt

```diff
@@ -0,0 +1,4 @@
+package org.jetbrains.dokka.utilities
+
+inline fun <reified T : Enum<*>> enumValueOrNull(name: String): T? =
```

Comment:
This should IMHO do the `toUppercase()`. Otherwise you'll have to repeat it everywhere and if the naming changes, we're doomed